### PR TITLE
fix Kubernetes blog

### LINKED ISSUE #18889: Two blog posts from early 2016 are corrupted
**This is a Bug Report**

**Problem:**
This post comes up in Chinese: https://kubernetes.io/blog/2016/01/why-kubernetes-doesnt-use-libnetwork/
This post shows with every character interspersed with "https://github.com/kubernetes/kubernetes/pull/18215": https://kubernetes.io/blog/2016/01/kubernetes-community-meeting-notes_28/

**Proposed Solution:**
Make the posts readable in English.

**Page to Update:**
I imagine this needs to be fixed in transformation; the source looks ok.

### PR DESCRIPTION
**Pull request by neha-viswanathan**
Closes #18889

### CODE REVIEW

**Review by irvifa — COMMENTED** [review]
/lgtm
/approve

File: content/en/blog/_posts/2016-01-00-Kubernetes-Community-Meeting-Notes_28.md [hunk 1]
```diff
@@ -8,58 +8,38 @@ url: /blog/2016/01/Kubernetes-Community-Meeting-Notes_28
...
-    - Quinton's Ubernetes Talk: &nbsp;https://youtu.be/L2ZK24JojB4
-    - Design for Ubernetes: https://github.com/kubernetes/kubernetes/pull/19313
-
-
-  - Conformance testing Q+A [Isaac Hollander McCreery]
-
+  - Use Case (10 min): SFDC Paul Brown(https://docs.google.com/a/google.com/presentation/d/1MEI97efplr3f-GDX1GcWGfkEuGKKV-4niu27kHOeMLk/edit?usp=sharing_eid&ts=56a114f8)
```
Comment (bboreham) [inline content/en/blog/_posts/2016-01-00-Kubernetes-Community-Meeting-Notes_28.md:11]: Suggest square brackets should have been left round "SFDC Paul Brown" to make a more readable link.
Same thing has happened multiple times in this PR.

File: content/en/blog/_posts/2016-01-00-Kubernetes-Community-Meeting-Notes_28.md [hunk 2]
```diff
@@ -8,58 +8,38 @@ url: /blog/2016/01/Kubernetes-Community-Meeting-Notes_28
...
-
+To get involved in the Kubernetes community consider joining our Slack channel(http://slack.k8s.io/), taking a look at the Kubernetes project(https://github.com/kubernetes/) on GitHub, or join the Kubernetes-dev Google group(https://groups.google.com/forum/#!forum/kubernetes-dev). If you’re really excited, you can do all of the above and join us for the next community conversation -- January 27th, 2016. Please add yourself or a topic you want to know about to the agenda(https://docs.google.com/document/d/1VQDIAB0OqiSjIHI8AWMvSdceWhnz56jNpZrLs6o7NJY/edit) and get a calendar invitation by joining this group(https://groups.google.com/forum/#!forum/kubernetes-community-video-chat).
```
Comment (bboreham) [inline content/en/blog/_posts/2016-01-00-Kubernetes-Community-Meeting-Notes_28.md:42]: Lost square brackets round "Slack Channel", "Kubernetes project", "Kubernetes-dev Google group", "agenda", "this group"

File: content/en/blog/_posts/2016-01-00-Kubernetes-Community-Meeting-Notes_28.md [hunk 3]
```diff
@@ -8,58 +8,38 @@ url: /blog/2016/01/Kubernetes-Community-Meeting-Notes_28
...
 
 
-Still want more Kubernetes? Check out the [recording](https://www.youtube.com/watch?v=izQLFx_6kwY&feature=youtu.be&list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ) of this meeting and the growing of the archive of [Kubernetes Community Meetings](https://www.youtube.com/playlist?list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ).
+Still want more Kubernetes? Check out the recording(https://www.youtube.com/watch?v=izQLFx_6kwY&feature=youtu.be&list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ) of this meeting and the growing of the archive of Kubernetes Community Meetings(https://www.youtube.com/playlist?list=PL69nYSiGNLP1pkHsbPjzAewvMgGUpkCnJ).
```
Comment (bboreham) [inline content/en/blog/_posts/2016-01-00-Kubernetes-Community-Meeting-Notes_28.md:45]: Lost square brackets round "recording", "Kubernetes Community Meetings"

**Pull request merged**